lib/modules/versioning/lambda-node/index.ts

```diff
@@ -0,0 +1,43 @@
+import { DateTime } from 'luxon';
+import {
+  getNewValue,
+  getSatisfyingVersion,
+  isValid,
+  matches,
+  minSatisfyingVersion,
+} from '../node';
+import npm from '../npm';
+import type { VersioningApi } from '../types';
+import { findLambdaScheduleForVersion } from './schedule';
+
+export const id = 'lambda-node';
+export const displayName = 'Lambda Node.js Runtime';
+export const urls = [];
+export const supportsRanges = false;
+
+export function isStable(version: string): boolean {
+  const schedule = findLambdaScheduleForVersion(version);
+
+  if (schedule === null) {
+    return false;
+  }
+
+  if (typeof schedule.support === 'string') {
+    return DateTime.now() < DateTime.fromISO(schedule.support);
```

Comment:
```suggestion
    return DateTime.local() < DateTime.fromISO(schedule.support);
```

use local time for comparison like we do in node versioning